### ISSUE TITLE
Concerns over very large Webpack module

### ISSUE THREAD

**Issue by NotNite** [bluesky-social/social-app#6605]
While working on [a Bluesky client mod](https://github.com/NotNite/nitesky), I kept on having modules resolve to ID `61215`. This seemed strange, so I looked into it and found out that **a majority of the site is being bundled into a singular Webpack module**!

I'm not an expert on Webpack so I can't say how this impacts performance, but it certainly doesn't seem like intended behavior. I'm unsure if this is caused by a Bluesky app config or something upstream in Expo.

There are roughly ~1000 modules total, some of which seem to be lazy loaded. Opening the entrypoint chunk in a text editor shows the modules (formatted with Prettier):

```js
(() => {
  var e = {
      61215: (e, t, r) => {
        // omitted for brevity
      },
      46580: (e, t, r) => {
        "use strict";
        r.r(t), r.d(t, { default: () => n });
        const n = crypto;
      },
      54993: () => {
        window.__BUNDLE_START_TIME__ = performance.now();
      },
      27124: () => {},
      57761: () => {},
      32559: (e, t, r) => {
        "use strict";
        e.exports =
          r.p + "static/media/InterVariable-Italic.55d6a3f35e9b605ba6f4.ttf";
      },
      92719: (e, t, r) => {
        "use strict";
        e.exports = r.p + "static/media/InterVariable.c9f788f6e7ebaec75d7c.ttf";
      },
      99044: (e, t, r) => {
        "use strict";
        e.exports = r.p + "static/media/kawaii.845d3efa31b8cd01ad2c.png";
      },
      51805: (e, t, r) => {
        "use strict";
        e.exports = r.p + "static/media/kawaii_smol.b3d15ded6ae36409ca18.png";
      },
      46794: (e, t, r) => {
        "use strict";
        e.exports = r.p + "static/media/logo.755f93713675760cb738.png";
      },
      24654: () => {},
      36429: () => {},
      29275: () => {},
    },
    t = {};
```

Build info: `Build version: 1.94.0; Bundle info: 3c30bb1 (prod); Bundle date: 24111623; Platform: web; Platform version: undefined` with script `main.d1fb63af.js`.

**Comment by leafwind** [comment]
@NotNite So I've noticed this, and I've worked on it for weeks, and I tried different approaches on optimizing the package.

TL;DR, it's not worth it from pagespeed insight's perspective.

Even you breakdown the huge webpack to small pieces, the performance won't change a lot.

A few reasons contributing it:
1. Browser has cache, so only the 1st time opening this app will be affected.
2. The bottleneck is not the fat webpack, but LCP. (Largest Contentful Paint)
3. Modules are already lazying loading (you can also force it in the code, but didn't help a lot)
